siliconcompiler/report/streamlit_viewer.py

```diff
@@ -575,12 +576,63 @@ def show_title_and_runs(title_col_width=0.7):
         streamlit.columns([title_col_width, 1 - title_col_width], gap="large")
 
     with title_col:
-        streamlit.title(f'{new_chip.design} dashboard', anchor=False)
+        streamlit.markdown(
+            '''
+            <head>
+                <style>
+                    .logo-container {
+                        display: flex;
+                        align-items: flex-start;
+                    }
+                    .logo-image {
+                        margin-right: 10px;
+                        margin-top: -10px;
+                    }
+                    .logo-text {
+                        display: flex;
+                        flex-direction: column;
+                        margin-top: -20px;
+                    }
+                    .text1 {
+                        color: #F1C437; /* Yellow color */
+                        font: IBM Plex Sans !important;
+                        font-weight:700 !important;
+                        font-size:30px !important;
+                        margin-bottom: -16px;
+                    }
+                    .text2 {
+                        color: #1D4482; /* Blue color */
+                        font: IBM Plex Sans !important;
+                        font-weight:700 !important;
+                        font-size:30px !important;
+                    }
+                </style>
+            </head>''',
+            unsafe_allow_html=True
+        )
+
+        streamlit.markdown(
+            f'''
+            <body>
+                <div class="logo-container">
+                    <img src="data:image/png;base64,{base64.b64encode(open(sc_logo_path,
+                    "rb").read()).decode()}" alt="Logo Image" class="logo-image"
+                    width="62" height="61">
```

Comment:
if you just specify the height, the width will be scaled correctly, no need to show both.